### ISSUE TITLE
Bug when using quotes in template string

### ISSUE THREAD

**Issue by neoziro** [pedronauck/docz#312]
## Bug Report

This code is not parsed by docz:

```mdx
---
name: Utilities
menu: 1. Basics
order: 6
---

import { Playground } from 'docz'

<Playground>
  {() => {
    const X = `${console.log('hello')}`
    return null
  }}
</Playground>
```

<img width="1440" alt="screen shot 2018-09-08 at 10 25 12" src="https://user-images.githubusercontent.com/266302/45252099-bd08e600-b351-11e8-8333-d6398fb46fef.png">

**To Reproduce**

Copy paste code in docz.

**Environment**

* OS: OSX 10.13.6
* Node version: v8.11.4
* Docz version: v0.11.1

**Comment by neoziro** [comment]
<img width="331" alt="screen shot 2018-09-08 at 10 30 43" src="https://user-images.githubusercontent.com/266302/45252129-44565980-b352-11e8-8f14-53fa468b19aa.png">

I think the problem is that you are not escaping `${}`. So template strings are broken.

**Comment by kennetpostigo** [comment]
I ran into this as well

**Comment by pedronauck** [comment]
Fixed on [v0.13.5](https://github.com/pedronauck/docz/releases/tag/v0.13.5) ✅
